notification/src/main/resources/config.liqubase/changelog/00001_init_new_order.xml

```diff
@@ -12,7 +12,6 @@
             </not>
         </preConditions>
         <comment>Table client exists</comment>
-
```

Comment:
Liquibase calculate hash of the whole changeset to verify if changeset already was executed and compare hash by changeset id at DATABASECHANGELOG table. You will have an error during app startup, thats why change of the already merged changesets is a bad practice.